atomic_defi_design/Dex/Constants/General.qml

```diff
@@ -592,7 +592,7 @@ QtObject {
         if (prevent_coin_disabling.running)
             return false
 
-        if (ticker === atomic_app_primary_coin || ticker === atomic_app_secondary_coin) return false
+        if (ticker === atomic_app_primary_coin || ticker === atomic_app_secondary_coin || ticker === atomic_app_third_coin) return false
```

Comment:
I don't think it's a good approach to hardcode "default" coins. What if we'll want to have 10 or 20 activated on first user login?

Reply:
We can make it a list. The 1st and 2nd are used to initiate base/rel so they need to be their own variable too. Avtual coins are defined in build metadata  (to allow for differences in whitelabels)

Reply:
I've tweaked it so adding coins to `g_default_coins` will ensure they are auto activated on login. For demonstration purposes, https://github.com/KomodoPlatform/atomicDEX-Desktop/pull/2071/commits/720660aad68a82bc8173c83c827ca695dffb811c includes RICK &  MORTY in this list, but will remove before final review/merge